workshops/checklists/lead_instructor.html

```diff
@@ -29,9 +29,10 @@
   <ul>
     <li>
       Create the
-      <a href="{{site.github_url}}/bc">GitHub repository</a>
-      that will host the workshop's web pages.
-      <a href="https://vimeo.com/87241285">This video</a> shows how to do it as of early 2014.
+      <a href="{{site.github_url}}/workshop-template">GitHub repository</a>
+      that will host the workshop's web pages, following <a
+          href="{{site.github_url}}/workshop-template/blob/gh-pages/README.md#creating-a-repository">these
+          instructions in the README.md file.</a>.
```

Comment:
Good catch, but the second link can be simplified to {{site.github_url}}/workshop-template/#creating-a-repository